src/main/java/com/mstar/paymentgateway/payoneer/PayoneerGateway.java

```diff
@@ -0,0 +1,72 @@
+package com.mstar.paymentgateway.payoneer;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PayoneerGateway {
+    private static final String serviceUrl = "https://api.sandbox.oscato.com/api/lists";
+
+    public PayoneerGateway() {
+    }
+
+    public static String getListUrl(String amount) throws IOException {
+        Map<String, String> customer = new HashMap<>();
+        customer.put("number", "42");
+        customer.put("email", "john.doe@example.com");
+
+        Map<String, String> payment = new HashMap<>();
+        payment.put("amount", amount);
+        payment.put("currency", "EUR");
+        payment.put("reference", "Shop 101 20-03-2016");
+
+        Map<String, String> style = new HashMap<>();
+        style.put("language", "en_US");
+
+        Map<String, String> callback = new HashMap<>();
+        callback.put("appId", "com.payoneer.checkout.examplecheckout.sdk");
+        callback.put("notificationUrl", "https://dev.oscato.com/shop/notify.html");
+
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put("transactionId", "tr101");
+        parameters.put("integration", "MOBILE_NATIVE");
+        parameters.put("channel", "MOBILE_ORDER");
+        parameters.put("country", "GB");
+        parameters.put("customer", customer);
+        parameters.put("payment", payment);
+        parameters.put("style", style);
+        parameters.put("callback", callback);
+        String auth = "APPLAUSE" + ":" + "jr55qongujj4h4ge58t896jeug67le8rstfmta9t";
```

Comment:
Authentication header values should be hidden or obfuscated. It is better and more secure to keep those properties in `application-{prod|dev}.properties` files with values mapped to Heroku config variables.

Reply:
Drop the token from the corresponding sandbox, generate a new token and move those properties to `application-{prod|dev}.properties` files as suggested.

Reply:
Resolved with #13